test/create/controller.spec.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+var expect  = require('chai').expect;
+var Angular = require('angular');
+var Sinon   = require('sinon');
+
+require('angular-mocks');
```

Comment:
the onboarding docs might be outdated, but just leaving this here for info, but we've moved away from using `mocks` in our angular service (Dashboard) in favor of `stub`s